functions.php

```diff
@@ -524,16 +524,52 @@ function twenty_twenty_one_non_latin_languages() {
  * @return void
  */
 function twentytwentyone_customize_preview_init() {
+	wp_enqueue_script(
+		'twentytwentyone-customize-helpers',
+		get_theme_file_uri( '/assets/js/customize-helpers.js' ),
+		array(),
+		get_theme_file_path( 'assets/js/customize-helpers.js' ),
+		true
+	);
+
 	wp_enqueue_script(
 		'twentytwentyone-customize-preview',
 		get_theme_file_uri( '/assets/js/customize-preview.js' ),
-		array( 'customize-preview', 'customize-selective-refresh', 'jquery' ),
+		array( 'customize-preview', 'customize-selective-refresh', 'jquery', 'twentytwentyone-customize-helpers' ),
 		get_theme_file_path( 'assets/js/customize-preview.js' ),
 		true
 	);
 }
 add_action( 'customize_preview_init', 'twentytwentyone_customize_preview_init' );
 
+
+/**
+ * Enqueue scripts for the customizer.
+ *
+ * @since 1.0.0
+ *
+ * @return void
+ */
+function twentytwentyone_customize_controls_enqueue_scripts() {
+
+	wp_enqueue_script(
+		'twentytwentyone-customize-helpers',
+		get_theme_file_uri( '/assets/js/customize-helpers.js' ),
+		array(),
+		get_theme_file_path( 'assets/js/customize-helpers.js' ),
```

Comment:
Same version issue here, and in the `wp_enqueue_script` below.

Reply:
fixed